Vwitter/View Controllers/SearchUsersViewController.m

```diff
@@ -0,0 +1,92 @@
+//
+//  SearchUsersViewController.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/8/22.
+//
+
+#import "SearchUsersViewController.h"
+#import "UserCell.h"
+#import <Parse/Parse.h>
+
+@interface SearchUsersViewController () <UITableViewDataSource, UITableViewDelegate, UISearchBarDelegate>
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+
+@property (strong, nonatomic) NSMutableArray *arrayOfUsers;
```

Comment:
nit: I would recommend renaming `filteredData` to be more specific, `filteredSearchResults` might be better. 

Also, for collection types like `NSArray`, `NSSet`, `NSDictionary` etc, it is a good idea to use "generics", which let you tell the compiler what kind of object the collection/array can contain, and will produce a compiler error if there is some edge case in the code where you are adding something other than that supported type. Compiler errors are way better than runtime errors, because compiler errors just give you a headache whereas runtime errors give users a headache

```
@property (strong, nonatomic) NSMutableArray<VWUser *> *arrayOfUsers;
@property (strong, nonatomic) NSArray<VWUser *> *filteredSearchResults;
```